src/pages/NotificationsPage.vue

```diff
@@ -268,17 +279,9 @@ export default {
       packageNotis: []
     }
   },
-  created() {
-    if (this.$serverGet != null) {
-      this.notis = []
-      this.getNotificationsAndUpdateSeen()
-      this.loadPackageNotification(this.packages)
```

Comment:
Is the removal of `this.loadPackageNotification(this.packages)` intentional? I understood you capturing `this.getNotificationsAndUpdateSeen()` this change but not loadPackageNotifications. I hope this was not removed by mistake.